src/app/content/highlights/components/Card.tsx

```diff
@@ -78,7 +81,17 @@ const Card = (props: Props) => {
     return null;
   }
 
-  const onRemove = () => props.data && props.remove(props.data.id);
+  const chapter = getCurrentChapter(sections, page);
```

Comment:
I've resigned from this and just used `getHighlightLocationForPage`

Reply:
on nomenclature, highlightLocation still doesn't describe what we're talking about. what it is is a highlight filter option id, where filter options can be either colors or book parts (chapter or page) `getHighlightFilterForPage` might make sense, but is potentially confusing because you don't know if its a color or location filer, which is why i proposed `highlightLocationFilter` so in this case `getHighlightLocationFilterForPage`, since you're going to be importing it from some `highlights/utils.ts` file, maybe just `getLocationFilterForPage` would be adequate